lighthouse-core/lib/lantern-trace-saver.js

```diff
@@ -53,13 +53,35 @@ function convertNodeTimingsToTrace(nodeTimings) {
   return {traceEvents};
 
   /**
-   * @return {LH.TraceEvent}
+   * @return {LH.TraceEvent.TracingStartedInPage}
    */
-  function createFakeTracingStartedEvent() {
+  function createFakeTracingStartedInPageEvent() {
     const argsData = {
       frameTreeNodeId: 1,
       sessionId: '1.1',
```

Comment:
what are these properties: `frameTreeNodeId`, `sessionId`, `persistentIds` ? only set here.

Reply:
at one point in time the tracing started events looked like this, or some amalgamation thereof :)

I think we're on the 3rd iteration of chrome tracing format support though, so I don't really remember